### ISSUE TITLE
TypeError: _reactDnd.DragDropContext is not a function when running example code

### ISSUE THREAD

**Issue by razvan100** [react-tags/react-tags#637]
I am trying to run the example provided with this library and importing it like

`import { WithContext as ReactTags } from 'react-tag-input';`
 
(following the **Usage** section from https://www.npmjs.com/package/react-tag-input)

However I'm getting the below error

```
TypeError: _reactDnd.DragDropContext is not a function
./node_modules/react-tag-input/dist-modules/components/ReactTags.js
node_modules/react-tag-input/dist-modules/components/ReactTags.js:538

  535 | };
  536 | 
  537 | module.exports = {
> 538 |   WithContext: (0, _reactDnd.DragDropContext)(_reactDndHtml5Backend2.default)(ReactTags),
  539 |   WithOutContext: ReactTags,
  540 |   KEYS: _constants.KEYS
  541 | };
```

  Found a link in a issue ticket, tells this is deprecated
  https://github.com/react-dnd/react-dnd/issues/1489#issuecomment-515033724

  My dependencies

    "react": "^16.13.1",
    "react-dnd": "^11.1.3",
    "react-dnd-html5-backend": "^11.1.3",
    "react-dom": "^16.13.1",
    "react-tag-input": "^6.4.3",

**Comment by orszaczky** [comment]
related to https://github.com/react-dnd/react-dnd/issues/1513 and https://github.com/react-dnd/react-dnd/issues/1489

**Comment by orszaczky** [comment]
also duplicate of https://github.com/prakhar1989/react-tags/issues/547

**Comment by Ayush909** [comment]
> I am trying to run the example provided with this library and importing it like
> 
> `import { WithContext as ReactTags } from 'react-tag-input';`
> 
> (following the **Usage** section from https://www.npmjs.com/package/react-tag-input)
> 
> However I'm getting the below error
> 
> ```
> TypeError: _reactDnd.DragDropContext is not a function
> ./node_modules/react-tag-input/dist-modules/components/ReactTags.js
> node_modules/react-tag-input/dist-modules/components/ReactTags.js:538
> 
>   535 | };
>   536 | 
>   537 | module.exports = {
> > 538 |   WithContext: (0, _reactDnd.DragDropContext)(_reactDndHtml5Backend2.default)(ReactTags),
>   539 |   WithOutContext: ReactTags,
>   540 |   KEYS: _constants.KEYS
>   541 | };
> ```
> 
> Found a link in a issue ticket, tells this is deprecated
> [react-dnd/react-dnd#1489 (comment)](https://github.com/react-dnd/react-dnd/issues/1489#issuecomment-515033724)
> 
> My dependencies
> 
> ```
> "react": "^16.13.1",
> "react-dnd": "^11.1.3",
> "react-dnd-html5-backend": "^11.1.3",
> "react-dom": "^16.13.1",
> "react-tag-input": "^6.4.3",
> ```

did you solve this issue? I am running into the same problem.

**Comment by razvan100** [comment]
Nope, I believe it was not solved yet.

**Comment by bruno-barros** [comment]
After v9, DnD changed to the Context API and broke this plugin implementation.
I solved setting DnD to v8.
```
"react-dnd": "8.*",
"react-dnd-html5-backend": "8.*",
```

**Comment by saiteja-konda** [comment]
exact this is working for me  react-dnd@8.0.0 react-dnd-html5-backend@8.0.0 @bruno-barros thanks you

**Comment by ebisLab** [comment]
v 8 worked for me, but that sucks. Can we get maintainers of this dependency to get this up to date?

**Comment by ad1992** [comment]
fixed via #729
